new-website/src/intl/en.json

```diff
@@ -0,0 +1,4 @@
+{
+    "pageTitle": "ImmunHeroes",
+    "pageDescription": "We connect people that recovered from Covid-19 and thus probably immune helpers with all those who need support"
```

Comment:
Nice! Out of interest: How would it work with markdown files?

Reply:
Just added the fix for the markdown files. Was not as easy since gatsby-plugin-intl does not support markdown directly ... :( but it works 🥳

Reply:
Yey